src/main/resources/application.properties

```
@@ -0,0 +1 @@
+spring.application.name=SWSInitialDemo
```

Do we need this? If no, I would remove it. If yes, then I would rename it:
```suggestion
spring.application.name=Sliding Work Sharing Demo Application
```